recipe/meta.yaml

```diff
@@ -1,14 +1,14 @@
 {% set python_min = "3.10" %}
```

Comment:
Yep, looks like we need to update the minimal reqs

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1285500&view=logs&j=7b6f2c87-f3a7-5133-8d84-7c03a75d9dfc&t=9eb77fd2-8ddd-5444-8fc0-71cb28dcb736

Reply:
So this now passes simply by bumping up the Python version requirement.

Do we also want to bump the upstream dep requirements or leave off for user flexibility?

@sjsrey @knaaptime @martinfleis